src/main/java/com/edmunds/rest/databricks/service/ClusterService.java

```diff
@@ -115,4 +117,15 @@ String create(CreateClusterRequest createClusterRequest)
    */
   ClusterInfoDTO[] list() throws IOException, DatabricksRestException;
 
+  /**
+   * Lists events of a specific cluster on a given databricks instance.
+   * Allows you to filter which events you want to see too.
+   * @see <a href="https://docs.databricks.com/api/latest/clusters.html#events">https://docs.databricks.com/api/latest/clusters.html#events</a>
+   * @return an array of cluster events objects
+   * @throws IOException any other errors
+   * @throws DatabricksRestException any errors with the request
+   */
+  ClusterEventsDTO listEvents(String clusterId, ClusterEventTypeDTO[] eventsToFilter,
```

Comment:
I understand the need for `ClusterEventsDTO` internally, but would clients of this service prefer a `List<ClusterEventDTO>` return type instead?